pkg/domainevent/domain/message_test.go

```diff
@@ -15,16 +15,23 @@
 package domain
 
 import (
+	"context"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc/metadata"
 
 	"github.com/bucketeer-io/bucketeer/pkg/locale"
 	proto "github.com/bucketeer-io/bucketeer/proto/event/domain"
 )
 
 func TestLocalizedMessage(t *testing.T) {
 	t.Parallel()
+	ctx := context.TODO()
+	ctx = metadata.NewIncomingContext(ctx, metadata.MD{
+		"accept-language": []string{"ja"},
```

Comment:
@ono-max 
Is it possible for you to use English instead?
The rest looks good!

Reply:
If there is a test checking the `accept-language` for `en` and `js`, it's ok as it is. Can you confirm?

Reply:
Talked at discord